BMM.Tests/BMM.UITests/AutoFollowFromKareTest.cs

```diff
@@ -36,7 +36,8 @@ public async Task Assert_AutoFollowFromKareWorks()
             await _bmmApp.LoginToApp();
             _bmmApp.Menu.OpenProfilePage(_app);
             _app.Tap(_bmmApp.SettingsPage.DownloadViaMobileNetworkSwitch);
-            _bmmApp.Menu.OpenLibrary(_app);
+            _bmmApp.Menu.OpenBrowse(_app);
+            _app.ScrollDownTo(Kare);
```

Comment:
in this test it works?
But in SettingsTest we needed to go through the home screen?
The Xamarin UI tests are really annoying

Reply:
Well, not exactly. This test has this weird try..catch block, and the test is ignored, if it fails...
So on appcenter the test looks like passed, but in the details we see:

![image](https://user-images.githubusercontent.com/19508142/142462294-082793b6-7420-435b-b420-c2aacfa99f71.png)

I try to change it to go to Fra Kaare through Explore page.